noxfile.py

```diff
@@ -97,7 +97,9 @@ def black(session: nox.Session) -> None:
     if session.posargs:
         session.run("python", "-m", "black", *session.posargs)
     else:
-        session.run("python", "-m", "black", "--diff", *DEFAULT_LINT_ARGUMENTS)
+        session.run(
```

Comment:
Is the `black` session supposed to *reformat* files, or to *check* for correct formatting? Depending on whether session arguments are provided this will do different things. Instead of adding `--check`, removing `--diff` might be a better solution.